src/ui/qgsadvanceddigitizingfloaterbase.ui

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>QgsAdvancedDigitizingFloaterBase</class>
+ <widget class="QWidget" name="QgsAdvancedDigitizingFloaterBase">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>128</width>
+    <height>77</height>
+   </rect>
+  </property>
+  <layout class="QGridLayout" name="gridLayout">
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
+   <property name="horizontalSpacing">
+    <number>6</number>
+   </property>
+   <property name="verticalSpacing">
+    <number>2</number>
+   </property>
+   <item row="3" column="1">
+    <widget class="QLabel" name="mAngleLabel">
+     <property name="text">
+      <string>Δa</string>
+     </property>
+    </widget>
+   </item>
+   <item row="2" column="3">
+    <widget class="QLineEdit" name="mDistanceLineEdit">
+     <property name="maximumSize">
+      <size>
+       <width>16777215</width>
+       <height>14</height>
+      </size>
+     </property>
+     <property name="font">
+      <font>
+       <pointsize>8</pointsize>
```

Comment:
The distance is the only lineedit with a fixed size, which results in a smaller font size here (hidpi screen) than the other floaters.

Reply:
Ok

Reply:
Has that been changed?

Reply:
Yes in d65e63f

Reply:
Isn't 8 very small (smaller than the rest of the fonts)? I'm a bit afraid for visually impaired users.

Reply:
Here (non-hidpi screen) it's actually the default it seems ? I'll remove the setting so it would display at the same size than other UI elements, and adapt with font size setting (in QGIS settings).